### ISSUE TITLE
Toggle for second item in /pricing is not working

### ISSUE THREAD

**Issue by ilarioni** [anny-co/tailwindcss-gutters#1]
I know it's not appropriate repo for addressing this issue, but anyways throw this here.
On following page - https://anny.co/en/pricing - second item in FAQ section - ''Can I upgrade or change my plan myself'' is not toggling.
I am just curious, if you will find the solution, please let me know :)

**Comment by zoomoid** [comment]
You are right, this is not an appropriate repository for such an issue, mostly because the code contained here is no longer used on the website itself and additionally is no longer maintained and/or needed - newer versions of tailwind offer better utility with regards to `space` and `gap`. 

Please direct your concerns directly to the team at anny, via email or otherwise available contact methods on the website.

@fibis you might want to archive this repository to prevent such issues in the future.
